searx/settings.yml

```diff
@@ -407,7 +407,7 @@ engines:
 
   - name : library genesis
     engine : xpath
-    search_url : http://libgen.io/search.php?req={query}
+    search_url : https://libgen.is/search.php?req={query}
```

Comment:
With httpS I got exception bellow:

    $ curl https://libgen.is/search.php
    curl: (60) SSL certificate problem: certificate has expired
    
````
Traceback (most recent call last):
...  File "~/searx/local/py3/lib/python3.7/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='libgen.io', port=80): Max retries exceeded with url: /search.php?req=foo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb0f43f0320>: Failed to establish a new connection: [Errno -2] Name or service not known'))
````

The only adress that worked for me is http://libgen.is/search.php

Reply:
That's expected, their certificate has been expired on 23th december and server isn't configured properly to auto-renew Lets Encrypt certification in every 3 months.

> The only adress that worked for me is http://libgen.is/search.php

In my humble opinion, we shouldn't use non-https version (i.e. http://libgen.is) if https version is available, we should wait until they don't generate a new certificate. @return42 what do you think?

Reply:
Is it possible to add two `search_url` addresses? Such that if first one didn't work, fallback to second `search_url` address.

Reply:
It's always better to use HTTPS instead of HTTP even if the certificate is expired/invalid.

Reply:
> In my humble opinion, we shouldn't use non-https version .. wait until they don't generate a new certificate. @return42 what do you think?

think same, please change to http://libgen.is and we can merge this PR

> Is it possible to add two search_url addresses? 

> It's always better to use HTTPS instead of HTTP even if the certificate is expired/invalid.

Both are new features / if you think any of has its value send PR or add issue for / thanks!

Reply:
> please change to http://libgen.is   

This is would be very very bad idea, anyone can clearly see all traffic between Searx instance and Libgen, can perfrom man-in-the-middle attack and all other disadvantages we've due using http instead of https. In my opinion, having a broken engine is much better solution than leaving all connection insecure because in few days admin will/must fix the problem, but once we added non-https URL here, we put every user in risk. If you still insist, I can change that to non-https, but I strongly oppose your of idea. Please think about it once.